Tests/test_content.py

```diff
@@ -317,55 +322,89 @@ def send_slack_message(slack, chanel, text, user_name, as_user):
+                   prints_manager, thread_index=0, is_mock_run=False):
+    if not tests_settings.is_private:
+        with acquire_test_lock(integrations,
+                               test_options.get('timeout'),
+                               prints_manager,
+                               thread_index,
+                               tests_settings) as lock:
+            if lock:
+                status, inc_id = test_integration(c, server_url, demisto_user, demisto_pass, integrations,
+                                                  playbook_id, prints_manager, test_options,
+                                                  is_mock_run, thread_index=thread_index)
+                # c.api_client.pool.close()
+                if status == PB_Status.COMPLETED:
+                    prints_manager.add_print_job('PASS: {} succeed'.format(test_message), print_color, thread_index,
+                                                 message_color=LOG_COLORS.GREEN)
+                    succeed_playbooks.append(playbook_id)
+
+                elif status == PB_Status.NOT_SUPPORTED_VERSION:
+                    not_supported_version_message = 'PASS: {} skipped - not supported version'.format(test_message)
+                    prints_manager.add_print_job(not_supported_version_message, print, thread_index)
+                    succeed_playbooks.append(playbook_id)
+
+                else:
+                    error_message = 'Failed: {} failed'.format(test_message)
+                    prints_manager.add_print_job(error_message, print_error, thread_index)
+                    playbook_id_with_mock = playbook_id
+                    if not is_mock_run:
+                        playbook_id_with_mock += " (Mock Disabled)"
+                    failed_playbooks.append(playbook_id_with_mock)
+                    if not tests_settings.is_local_run:
+                        notify_failed_test(slack, circle_ci, playbook_id, build_number, inc_id, server_url, build_name)
+
+                succeed = status in (PB_Status.COMPLETED, PB_Status.NOT_SUPPORTED_VERSION)
             else:
-                error_message = 'Failed: {} failed'.format(test_message)
-                prints_manager.add_print_job(error_message, print_error, thread_index)
-                playbook_id_with_mock = playbook_id
-                if not is_mock_run:
-                    playbook_id_with_mock += " (Mock Disabled)"
-                failed_playbooks.append(playbook_id_with_mock)
-                if not tests_settings.is_local_run:
-                    notify_failed_test(slack, circle_ci, playbook_id, build_number, inc_id, server_url, build_name)
-
-            succeed = status in (PB_Status.COMPLETED, PB_Status.NOT_SUPPORTED_VERSION)
+                tests_queue.put(conf_json_test_details)
+                succeed = False
+    else:
+        status, inc_id = test_integration(c, server_url, demisto_user, demisto_pass, integrations, playbook_id, prints_manager,
```

Comment:
why do we remove the lock mech from the private?

Reply:
It requires setting up the GCS client within the test content step which currently is done via env variables and setting up the client in a different way than the rest of the build. It's something we could add back at a later time but currently is high effort so we skipped it for now.

Reply:
let's open an issue for this please

Reply:
https://github.com/demisto/etc/issues/29567